hdf5-src/build.rs

```diff
@@ -33,6 +33,15 @@ fn main() {
     println!("cargo::rerun-if-changed=build.rs");
     let mut cfg = cmake::Config::new("ext/hdf5");
 
+    if cfg!(target_env = "msvc") {
+        if let Ok(var) = env::var("CMAKE_POLICY_DEFAULT_CMP0091") {
+            cfg.define("CMAKE_POLICY_DEFAULT_CMP0091", var);
+        }
```

Comment:
```suggestion
    if cfg!(target_env = "msvc") {
        cfg.define("CMAKE_POLICY_DEFAULT_CMP0091", var);
```
or is there a reason for not setting this policy?